--- qtwebkit-opensource-src-5.6.0/Source/JavaScriptCore/LLIntOffsetsExtractor.pro.0005~	2016-03-24 12:18:23.627303498 +0100
+++ qtwebkit-opensource-src-5.6.0/Source/JavaScriptCore/LLIntOffsetsExtractor.pro	2016-03-24 12:19:28.706770081 +0100
@@ -13,12 +13,7 @@ debug_and_release {
     CONFIG += build_all
 }
 
-# Don't try to link against any Qt libraries, but at least
-# pull in include paths as we include qglobal.h.
-INCLUDEPATH += $$QT.core.includes
-DEFINES += QT_NO_VERSION_TAGGING
-CONFIG += console
-CONFIG -= qt
+QT = core # Needed for qglobal.h
 
 defineTest(addIncludePaths) {
     # Just needed for include paths
