--- qtwebkit-opensource-src-5.9.0-alpha/Tools/qmake/mkspecs/features/default_pre.prf.omv~	2017-03-25 00:00:10.350962077 +0100
+++ qtwebkit-opensource-src-5.9.0-alpha/Tools/qmake/mkspecs/features/default_pre.prf	2017-03-25 00:00:20.759709576 +0100
@@ -62,10 +62,6 @@ win32-msvc*: MAKEFILE_NOOP_COMMAND = @ec
 scratchbox: PYTHON = python2.6
 else: PYTHON = python
 
-# We use private_tests to detect developer build, since the destdir will
-# always be our webkit build dir. This might change as configure changes.
-qtConfig(private_tests): CONFIG += qt_developer_build
-
 # By default we enable "production build", and build-webkit, which is
 # used by bots and developers, will disable it, to enable warnings etc.
 CONFIG += production_build
